ui/tests/acceptance/secrets/backend/kv/diff-test.js

```diff
@@ -28,7 +28,7 @@ module('Acceptance | kv2 diff view', function (hooks) {
     this.server.shutdown();
   });
 
-  test('it shows correct diff status based on versions', async function (assert) {
+  test.skip('it shows correct diff status based on versions', async function (assert) {
```

Comment:
Non-blocking/Nit: 

`import { skip } from 'qunit';`
and use `skip()`

Reply:
We already import test so I'd rather just use skip from that